shared/js/ui/views/whitelist.es6.js

```diff
@@ -99,7 +95,8 @@ Whitelist.prototype = window.$.extend({},
         [this.$remove, 'click', this._removeItem],
         [this.$add, 'click', this._addItem],
         [this.$showadd, 'click', this._showAddToWhitelistInput],
-        [this.$url, 'keyup', this._manageInputChange]
+        [this.$url, 'keyup', this._manageInputChange],
+        [this.store.subscribe, 'change:whitelist', this.rerender]
```

Comment:
Listening to the event published by the model - this will fire on load and when adding new elements